common/versioned_submit_block_request.go

```diff
@@ -168,3 +173,145 @@ func (r *VersionedSubmitBlockRequest) ExecutionPayloadLogsBloom() ([256]byte, er
 		return [256]byte{}, errors.New("unsupported version")
 	}
 }
+
+// UnmarshalSSZ ssz unmarshals the SubmitBlockRequest object
+func UnmarshalSSZFast(s *builderApiElectra.SubmitBlockRequest, buf []byte) error {
```

Comment:
Should this still exist? We have many versions of unmarshalSSZFast(s *builderApiElectra.SubmitBlockRequest) now

Reply:
agreed to move in next PR